### ISSUE TITLE
Gatsby/Docz build failures with micromodal

### ISSUE THREAD

**Issue by nsahlas** [ghosh/Micromodal#311]
We are using Docz and Gatsby as well as server side rendering. In certain instance the window is not yet available during server side rendering therefore our docz build fails. I would like to know if you would consider putting a wrapper around the window.micromodal code (around line 440):

// Wrap the require in check for window
if (typeof window !== `undefined`) {
  window.MicroModal = MicroModal;
}

This actually resolves the build errors.

**Comment by ghosh** [comment]
@nsahlas Thanks for reporting. SSR was not something we had considered till now. Will fix this in next release.

**Comment by nsahlas** [comment]
@ghosh  was this resolved with the last release?

**Comment by spasticninja** [comment]
+1 for this issue. We are server rendering and I'm getting the the window is undefined error.

**Comment by mumanity** [comment]
You give simple fix. 

We smile.

tl;dr
+1

**Comment by brianyuen** [comment]
I'm still getting the `"window" is not available during server side rendering.` error on `'gatsby build'.` Is there a way to fix this??

**Comment by benaltair** [comment]
In terms of SSR, I'm getting the same error on [Svelte Kit](https://kit.svelte.dev/).
```
ReferenceError: window is not defined
    at /node_modules/micromodal/dist/micromodal.es.js:437:1
    at instantiateModule (D:\Github\conference\node_modules\vite\dist\node\chunks\dep-66eb515d.js:69030:166)
```
Looks like Micromodal still isn't supporting SSR as it references the DOM on import. Perhaps the DOM reference can be changed to support SSR frameworks?

**Comment by ghosh** [comment]
Fixed in #443
